src/jupyterlabcontroller/config.py

```diff
@@ -322,6 +332,16 @@ class LabConfig(CamelCaseModel):
         ),
     )
 
+    @field_validator("homedir_prefix")
+    @classmethod
+    def _validate_homedir_prefix(cls, v: str) -> str:
+        return v.rstrip("/")
```

Comment:
Should this not check that the prefix is not "/"?  I don't know who would make user directories the root, but returning the empty string if they do probably isn't the right answer?